test/support/helpers.js

```diff
@@ -0,0 +1,7 @@
+exports.fetchProp = function (o, prop) {
```

Comment:
Please stop using this as we will be converting to TS soon

Reply:
@rjrodger Until we switch to TS, perhaps it would be of benefit to have that extra safety in place?